spec/latest/json-ld-framing/index.html

```diff
@@ -367,6 +367,31 @@ <h3>Framing</h3>
     -->
     </pre>
 
+    <p>If <a>processing mode</a> is <code>json-ld-1.1</code> or greater, or the <a>omit graph flag</a> is <code>true</code>,
```

Comment:
I think the default for the flag should be `false` given that so much code exists that expects it to be there.